README.md

```diff
@@ -49,13 +53,19 @@ Start chatting
 ## Usage
 
 ### Enter a username
-![username](./images/NameLogin.png)
+![username](./images/enterUsername.png)
 
 ### Type in your preferred language
-![Language](./images/LanguageChoice.png)
+![Language](./images/hello.png)
+
+### The last 5 messages from the chat will be displayed
```

Comment:
Chat history display has been changed. Can you update the image?

Reply:
I updated the image